scrnaseq.Rmd

```diff
@@ -1963,6 +1981,10 @@ To gain first insights into potential functions of cells in a cluster, we test f
 
 We first translate gene symbols of up- and down-regulated genes per cluster into Entrez gene symbols, and then use the "enrichR" R-package to access the "Enrichr" website `r Cite("https://amp.pharm.mssm.edu/Enrichr/", "citep")`. You can choose to test functional enrichment from a wide range of databases:
 ```{r enrichr_databases}
+# Set Enrichr database
+enrichR::setEnrichrSite(param$enrichr_site)
```

Comment:
Can you deactivate the enrichr messages? Maybe you can tell enrichr to not be verbose?

Reply:
I like the messages, so I know what database it connected to...